docs/source/interfaces/ILL/SimpleScanViewer.rst

```diff
@@ -0,0 +1,78 @@
+* reducing a file through the Advanced button, which opens the dialog for
+  :ref:`SANSILLParameterScan <algm-SANSILLParameterScan>` and allows for changing the default parameters.
+
+After the algorithm has run, the output is displayed in a slice viewer-like interface. On the right is a table that
+shows the coordinates of all the created ROIs and allows their editing.
+
+Some keyboard inputs allow exporting data to workspaces for further analysis (see below).
+
+
+Modes
+-----
+
+There 3 modes of interaction with the slice viewer interface :
+
+Move
+    Clicking and dragging will move the view around.
+Zoom
+    Clicking and dragging will zoom on the selected region. Right-clicking and dragging will de-zoom. Alternatively, in
+    all modes, using the mouse wheel will also zoom/de-zoom.
+ROI
+    Clicking and dragging will define a rectangular region of interest. Multiple such regions can be created, and they
+    are the main analysis tool provided. They can be selected, deselected, moved, and resized by clicking on the control
+    points.
+
+Features
+--------
+
+When a ROI is defined, the X and Y axis projections are plotted in the side plots attached to their respective axis.
+A small red cross is also drawn in the ROI, showing the position of the center of mass of this ROI (assumed to be a good
+enough approximation of the peak position).
+
+A number of keyboard shortcuts are provided to interact with the data and the ROIs defined.
+
+
+'Del'
+    Deletes the currently selected ROI.
+
+'x', 'y', 'c'
+    Export the cuts - the side plots - to workspaces. "x" exports the one along the X-axis, "y" the one along the Y-axis,
+    and "c" along both of them. A different workspace is created for every ROI on each axis, in addition to a workspace
+    containing the side plot itself (which is just the sum of all the other workspaces based on that axis).
+'p'
+    Exports the peaks to a table workspace. For each ROI, the detected peak is the center of mass of the ROI. The values
+    exported are the index of the rectangle, the coordinates of the peak in :math:`2\theta, \Omega`, the data integrated
+    over the ROI, and, if a background has been provided, the integration corrected by the background (i.e. integration
+    of the sample data minus the background data with minimal reduction). NB : this last value will not make sense if
+    the data shown in the data viewer has already been reduced through the `Advanced` button.
+'f'
+    Tries to extrapolate the ROIs currently drawn, with two behaviours:
+
+    * if there is only one ROI, it is assumed to correspond to the first non-zero theta peak and linearly place other
+      ROIs at regular intervals assuming :math:`\theta = \Omega`. No ROI is placed at :math:`\theta = 0`, because it is
+      the beam and does not contain data.
+    * if there are 2 ROIs, it simply linearly interpolates from these two to place more ROIs. Their size is the mean of
+      the original ROIs.
```

Comment:
The other case is not covered and will lead to no extrapolation which could be understood but perhaps it is worth a mention that if there are more than two ROIs, then the button does nothing. What do you think?